src/bp/ui-admin/src/telemetry/telemetry_type.d.ts

```diff
@@ -0,0 +1,34 @@
+export interface StoreInfoBody {
+  storedInfo: string
+  loadInfo: Function
+}
+
+export interface StoreInfoType {
+  [key: string]: StoreInfoBody
+}
+
+export interface EventPackageInfoType {
+  [key: string]: EventPackageInfoBody
+}
+
+export interface EventPackageInfoBody {
+  locked: boolean
+  timeout: string
+  getPackage: Function
+}
+
+export interface EventData {
```

Comment:
1st, don't export things that are not used anywhere else in the code

2nd this interface can simply be nested in the TelemetryPackage interface


```
export interface TelemetryPackage {
  schema: string
  uuid: string
  timestamp: string
  bp_release: string
  bp_license: string
  event_type: string
  source: string
  event_data: {
   schema: string
   [key:string] : any
 }
}

```

Reply:
Fixed in : e135fcf